connectors/airwatch/src/test/resources/application.properties

```diff
@@ -1 +0,0 @@
-security.oauth2.resource.jwt.key-uri=https://acme.vmwareidentity.com/SAAS/API/1.0/REST/auth/token?attribute=publicKey&format=pem
```

Comment:
I seem to remember needing this for the tests to pass.  Was this just memory of the old docker container tests? (The unit tests aren't hitting whatever code that asks for vIdmPubKey and are still passing?)

Reply:
Ah I see.  Not shown in this diff is the key-value is set in the connectors-test's properties file: https://github.com/vmware/connectors-workspace-one/blob/master/common/connectors-test/src/main/resources/application-test.properties